qemu/tests/cfg/virtio_fs_share_data.cfg

```diff
@@ -146,7 +146,14 @@
                     image_snapshot = yes
                     mem = 32768
                     size_mem1 = 32G
-                    blacklist = generic/551
+                    tmpfs_size = 32G
```

Comment:
Actually, the size of tmpfs depends on the size of host mem by default. In linux, it defaults to half of mem.
Like this:
```
[root@ampere-hr350a-04 ~]# df -h | grep /dev/shm | gawk '{ print $2 }'
128G
[root@ampere-hr350a-04 ~]# free -h
              total        used        free      shared  buff/cache   available
Mem:          254Gi       3.8Gi       240Gi        32Mi        11Gi       228Gi
Swap:         4.0Gi          0B       4.0Gi
```
So I think the bug error is that 'tmpfs' is too small because the host mem is less than 64G.
I think we should add a judgment to the code, when the host mem is less than 64G, manually set the tmpfs size.

Reply:
@YongxueHong what do you think?

Reply:
>  Actually, the size of tmpfs depends on the size of host mem by default. In linux, it defaults to half of mem.

Yes.
>  So I think the bug error is that 'tmpfs' is too small

Yes, 16G is not enough for this xfstest

>  when the host mem is less than 64G, manually set the tmpfs size.

Ok, i got it. If we need add a judgement why not check /dev/shm size directly 😃

Reply:
Oh yes, you're right.
But I'm not sure, if host `/dev/shm` too small, we set it to 32G, when the test is done, do we need to restore its original size?
@YongxueHong

Reply:
>  But I'm not sure, if host /dev/shm too small, we set it to 32G, when the test is done, do we need to restore its original size?
It already has the restore step in virtio_fs_share_data.py

Reply:
Yes, I mean if we don't need to recover, we can omit this step.

Reply:
After discussion with @YongxueHong
Since `mem-path_mem1 = /dev/shm` line 23, there is no need to add additional parameters, just use `size_mem1` as the minimum value.
On the other hand, the size needs to be restored after the test is done.

Reply:
ok, got it. Thanks a lot.